### PR TITLE
task 1 Apoorva kumar

### PR DESCRIPTION
**Pull request by apoorvakumar1998**

### CODE REVIEW

**Comment by anoopbhatn** [comment]
1. Provide a proper description stating the changes done in the PR
2. Include the task name in the PR title
3. Please don't create a new branch/PR for every change. Push your changes to the same branch/PR

File: sum.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sum=0;//variable to store sum
```
Comment (anoopbhatn) [inline sum.py]: Leave a comment block about the file created date and author something like this :
'''
Created on October 22, 2018
@author: anoopbhatn
'''

After this, specify the purpose of the code in a comment block

File: sum.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+start = int(input("Enter Start value:"))//inputing start value
```
Comment (anoopbhatn) [inline sum.py]: Provide the comment line above the line about which you will be writing a comment. Something like : 
\# inputing start value  
start = int(input("Enter Start value:"))

File: sum.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+sum=0;//variable to store sum
+start = int(input("Enter Start value:"))//inputing start value  
+end   = int(input("Enter End value:"))//inputing end value
+ for i in range(start,end+1)://for loop to determine sum
```
Comment (anoopbhatn) [inline sum.py]: Write a comment block stating the purpose

**Review by anoopbhatn — CHANGES_REQUESTED** [review]

File: sum.py [hunk 4]
```diff
@@ -1,6 +0,0 @@
-sum=0;//variable to store sum
```
Comment (anoopbhatn) [inline sum.py]: Remove this file and push

git rm sum.py

git commit -m "Task-2 : Removed sum.py as a new file sum_range.py replaces the functionality of that"

git push -u origin <branch_name>

File: sum_range.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+
+#initializing the variable to store sum
+sum=0;
+#inputing start value  
+start = int(input("Enter Start value:"))
+#inputing end value
+end   = int(input("Enter End value:"))
+#for loop to determine the sum of all integers in the range [A,B]
+ for i in range(start,end+1):
```
Comment (anoopbhatn) [inline sum_range.py]: Will this work? Won't this throw wrong indentation error?

File: sum_range.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+sum=sum + i
```
Comment (anoopbhatn) [inline sum_range.py]: Also this. This should be inside the for loop right?

**Review by anoopbhatn — CHANGES_REQUESTED** [review]

**Review by anoopbhatn — CHANGES_REQUESTED** [review]
The code looks good. Great job.
Can you put the file inside a folder named after you ?

**Comment by apoorvakumar1998** [comment]
how to do that?

**Comment by anoopbhatn** [comment]
Normal way. Just create a folder, put this code inside that. git add that folder. git rm the file outside and push.

**Pull request closed without merging**